docs/pages/careers/react-community-engineer-core.md

```diff
@@ -0,0 +1,130 @@
+# React Community Engineer — Material UI
+
+<p class="description">You will provide guidance to the community and solve their struggle, working on one of the most popular React UI library.</p>
+
+## Details of the role
+
+- **Location**: Remote (preference for UTC-6 to UTC+5).
+- **Type of work**: Full-time (contractor or employee [depending on circumstances](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501#494af1f358794028beb4b7697b5d3102)).
+- **Level**: [IC4 or above](https://mui-org.notion.site/Leveling-at-MUI-5c30f9bfe65149d697f346447cef9db1).
+- We're a **remote** company, we prefer asynchronous communication over meetings.
+
+## The company
+
+MUI's story began in 2014 with Material UI, the most successful React implementation of Google's Material Design.
+Today, Material UI stands as one of the most popular open-source libraries on GitHub and has paved the way for the fully-fledged startup known as MUI (founded in 2019), which now boasts an ever-expanding ecosystem of React UI products.
+We're a company of 35+ people as of early 2025, and we're growing.
+
+## The products
+
+MUI is best known for our flagship product, Material UI—but this is just one of three core component libraries we maintain.
+Base UI is our headless component library, and Pigment CSS is a zero-runtime CSS-in-JS styling engine.
+We also host Design Kits and pre-built Templates.
+
+Beyond the core libraries, MUI X offers advanced components like the Data Grid, Date and Time Pickers, and Charts, for more complex user interactions and data visualization needs.
+
+Learn more about MUI's products in this blog post: [An introduction to the MUI ecosystem](https://mui.com/blog/mui-product-comparison/).
+
+## The culture
+
+MUI is a fully remote company with a team that spans across the globe.
+The majority of our work is asynchronous, and we rely mostly on written communication to collaborate.
+We're radically transparent: nearly all of our work happens in public.
+Each contributor has the freedom to decide how and when they work, and that work is primarily self-directed: it's your responsibility to define and complete your own tasks in a timely manner.
+
+For additional details about the culture, you can check our [careers](https://mui.com/careers/) and [about](https://mui.com/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).
+
+## Why we're hiring
+
+We need a Community Engineer to solve friction points of users.
+We face challenges on multiple fronts:
+
+- As a result of this growth, we (will) have more users questions and feedback coming in than ever before.
```

Comment:
This is a bit confusing. What "growth" are you referring to?

Reply:
```suggestion
- As the library grows, we (will) have more users questions and feedback coming in than ever before.
```

Is it more clear?